math/caffe/Portfile

```diff
@@ -117,18 +125,18 @@ variant python27 description {Install Python 2.7 interface} {
                     port:py27-protobuf \
                     port:py27-gflags \
                     port:py27-leveldb \
-                    port:py27-dateutil
+                    port:py27-dateutil \
+                    port:py27-pydot
 
     build.target-append  pycaffe
+    test.target-append  pytest
```

Comment:
Little nitpick, you can add one more space before `pytest` to align it with the line above.

Reply:
👍